src/tests/health_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/routes"
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/tests/fixtures"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHealthRoute(t *testing.T) {
+	os.Setenv("env", "test")
+	env := os.Getenv("env")

```

Comment:
I think this should come from a test runner, then be set up in every test

Reply:
It will be handled by viper in future PR